app/models/area.rb

```diff
@@ -3,7 +3,7 @@ class Area < ApplicationRecord
   has_many :problems, through: :demands
   has_many :users
 
-  def short_name
-    self.name.split(",")[0].strip()
+  def to_i
```

Comment:
Didn't we realized  you didn't need to do this? That you could pass an arg to the `input` method for simple_form?

Reply:
this was still needed when returning to the form after tryign to submit with an error. not totally sure about it tbh.

Reply:
follow up on this: failed demand because no text but with multiple selected areas.. only one area comes back when form is rerendered.